editor/src/components/canvas/canvas-strategies/absolute-move-strategy.ts

```diff
@@ -38,27 +45,58 @@ export const absoluteMoveStrategy: CanvasStrategy = {
       interactionState.interactionData.drag != null
     ) {
       const drag = interactionState.interactionData.drag
-      return canvasState.selectedElements.flatMap((selectedElement) => [
-        adjustNumberProperty(
-          'permanent',
-          selectedElement,
-          stylePropPathMappingFn('left', ['style']),
-          drag.x,
-          true,
-        ),
-        adjustNumberProperty(
-          'permanent',
-          selectedElement,
-          stylePropPathMappingFn('top', ['style']),
-          drag.y,
-          true,
-        ),
+      return [
+        ...canvasState.selectedElements.flatMap((selectedElement) => {
+          const element: JSXElement | null = withUnderlyingTarget(
```

Comment:
This is just a code style remark, i think this is too much inlining here with the `return` and the spread out flatmap in it with a long inline function inside.
I would at least give the result of the flatmap a name, maybe even the mapping function could be named too.

Reply:
I improved it with 7c4e5bc6